packages/nx-plugin-openapi/src/executors/generate-api-lib-sources/schema.d.ts

```diff
@@ -1,6 +1,6 @@
-import { JsonObject } from '@angular-devkit/core';
+export interface BuildExecutorSchema {} // eslint-disable-line
```

Comment:
Why do we need this empty interface?
If there's no direct substitute for the old `JsonObject` I'd say let's not extend anything, or let's copy the old `JsonObject` somewhere and keep extending it. 😄

Reply:
I have removed the useless `BuildExecutorSchema` interface